x11-misc/synergy/synergy-1.7.5-r2.ebuild

```diff
@@ -113,6 +115,10 @@ pkg_preinst() {
 
 pkg_postinst() {
 	use qt4 && gnome2_icon_cache_update
+	einfo
+	einfo "To enable encryption for Pro users, link or copy"
```

Comment:
Please don't suggest copying as it will make it hard to update. Also, it would be a good idea to put a ready `ln` call for copy-paste instead.